history-of-quakers/modernized/07-chapter-5.adoc

```diff
@@ -763,13 +804,19 @@ hereof fail not at your peril.
 Given under my hand, and seal of Plymouth aforesaid, the 28th day of May,
 in the year of our Lord God, 1655.
 
-JOHN PAGE, Mayor.
+[.signed-section-signature]
+John Page, Mayor.
+
+--
 
 By this may be seen under what frivolous pretences those called Quakers were imprisoned,
 that is, because of an ordinance made against duels,
 etc. and as for their having refused to give security, how untrue this was,
 as well as other accusations, may appear from the following certificate.
 
+[.embedded-content-document.legal]
```

Comment:
Not sure if this should be labeled [legal].

Reply:
seems good to me 👍